tests/screenshot_generator/generator.py

```diff
@@ -16,7 +16,7 @@
 # These must precede any SeedSigner imports.
 sys.modules['seedsigner.hardware.displays.st7789_mpy'] = MagicMock()
 sys.modules['seedsigner.hardware.displays.ili9341'] = MagicMock()
-sys.modules['seedsigner.views.screensaver.ScreensaverScreen'] = MagicMock()
+sys.modules['seedsigner.gui.screens.ScreensaverScreen'] = MagicMock()
```

Comment:
Because `ScreensaverScreen` is now being called by `run_screen` in its View, it doesn't need to be mocked out. The screenshot generator already prevents any Screen from being instantiated in `run_screen`.